### ISSUE TITLE
Look into non-generic overloads for creating ops & non-generic JsonPatchDoc

### ISSUE THREAD

**Issue by KevinDockx** [KevinDockx/JsonPatch#64]
Might conflict with JsonPatch.Dynamic?

cfr https://github.com/KevinDockx/JsonPatch.Dynamic/issues/10#issuecomment-273648173

**Comment by HappyNomad** [comment]
The two non-generic `JsonPatchDocument` classes would be in different namespaces, so the "conflict" is manageable.

**Comment by KevinDockx** [comment]
Yep, but the thing is that we'd end up with two classes that are copies of each other but just live in a different namespace - and that's not right.  The best way to solve this from a design POV would be to move the non-generic JsonPatchDoc to this lib, and remove it from JsonPatch.Dynamic altogether, but that means a breaking change in JsonPatch.Dynamic.  That could be mitigated by marking it as obsolete in JsonPatch.Dynamic (and remove it altogether later on) - what do you think?

**Comment by HappyNomad** [comment]
That approach sounds fine to me.
